debvader/model.py

```diff
@@ -207,12 +215,18 @@ def load_deblender(
         experimental_run_tf_function=False,
     )
 
-    # Load the weights corresponding to the chosen survey
-    data_path = pkg_resources.resource_filename("debvader", "data/")
-    loading_path = os.path.join(data_path, "weights", survey, "not_normalised", "loss")
-    print(loading_path)
-    latest = tf.train.latest_checkpoint(loading_path)
-    net.load_weights(latest)
+    if external_path_to_folder is None:
+        # Load the weights corresponding to the chosen survey
+        data_path = pkg_resources.resource_filename("debvader", "data/")
+        loading_path = os.path.join(
+            data_path, "weights", survey, "not_normalised", "loss"
+        )
+        print(loading_path)
+        latest = tf.train.latest_checkpoint(loading_path)
+        net.load_weights(latest)
+    else:
+        latest = tf.train.latest_checkpoint(external_path_to_folder)
+        net.load_weights(latest)
```

Comment:
lot a code duplication here, could be avoided by taking these lines out of the if–else.
Something like 
```python
if external_path_to_folder is None:
    data_path = pkg_resources.resource_filename("debvader", "data/")
    external_path_to_folder = os.path.join(
            data_path, "weights", survey, "not_normalised", "loss"
        )
checkpoint = tf.train.latest_checkpoint(external_path_to_folder)
net.load_weights(checkpoint)
```